install/03_rbac.yaml

```diff
@@ -4,6 +4,16 @@ kind: ClusterRole
 metadata:
   name: cluster-autoscaler-operator
 rules:
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - validatingwebhookconfigurations
+  - mutatingwebhookconfigurations
+  verbs:
+  - get
+  - patch
```

Comment:
will the service-ca-operator patch or update here?

Reply:
I'm not sure what the service-ca-operator does when it updates the `caBundle`. In the interim solution where the cluster-autoscaler-operator itself copies the CA from a ConfigMap to the webhook config, we use a PATCH, so I've added these permissions.